src/DataStructures/DataBox.hpp

```diff
@@ -1040,11 +1135,11 @@ constexpr DataBox<TagsList<Tags...>>::DataBox(
       "The type of each Tag must be the same as the type being passed into "
       "the function creating the new DataBox.");
   // Create dependency graph between compute items and items being reset
-  using edge_list =
+  using local_edge_list =
```

Comment:
we assume the prepend of local_ is to avoid name conflicts?  I vaguely remember not liking local for this purpose in an earlier PR, but we don't remember why or what we had local changed to

Reply:
I remember this being an issue for template parameters, and I think the issue was changing `LArgs` -> `LocalArgs`. In many cases they were also `List` -> `Pack` so then the `Local` was no longer necessary. How about `kept_compute_item_edge_list`? It's a bit long but that is exactly what it is

Reply:
👍